fast/stages/1-resman/variables-fast.tf

```diff
@@ -56,8 +56,8 @@ variable "custom_roles" {
   type = object({
     gcve_network_admin              = string
     network_firewall_policies_admin = string
-    ngfw_enterprise_admin           = string
-    ngfw_enterprise_viewer          = string
+    ngfw_enterprise_admin           = optional(string)
+    ngfw_enterprise_viewer          = optional(string)
```

Comment:
is this `optional` ok?

Reply:
it's only used if you apply netsec, but we can make it mandatory

Reply:
actually let's make the other s3 (only gcve I think) roles optional too